tests/__fixtures__/tracking-plan-fixture.json

```diff
@@ -2,7 +2,7 @@
   "events": [
     {
       "description": "Don't do this.",
-      "name": "42_--terrible==event++name~!3",
+      "name": "42_--terrible==\"event'++name~!3",
```

Comment:
Discovered that we aren't properly escaping quotes, so I've added this to the test suite and fixed the JS/Android/iOS clients.